.github/workflows/ci.yml

```
@@ -45,7 +45,9 @@ jobs:
       - name: Setup .NET SDK
         uses: actions/setup-dotnet@v2
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 6.0.x |
+            7.0.x
```

The CI run is not starting because the `|` does not mean "there is more on the next line", it means "I intend to have an indented multi-line thing below this.
```suggestion
          dotnet-version: |
            7.0.x
            6.0.x
```

Though I am curious what would have happened if 6 had been specified before 7. Which SDK would be used to run `dotnet build`? We usually do highest first, assuming that the first one will be "primary". I'll test this on a repo that doesn't require multiple cumulative hours of CI time.